buildinfo.txt

```diff
@@ -1,4 +1,4 @@
-dependson=COWorkflowsBasePlugin:1.4.0
+dependson=COWorkflowsBasePlugin:1.4.1
```

Comment:
Only robustness-changes (related to checks for IO-errors). This version is already in use in recent Indel and should be unproblematic.